tests/unit/src/lib/settings.test.js

```diff
@@ -16,19 +16,21 @@ describe("settings", () => {
     it("should set 'API.API_BASE_URL", () => {
       setAPIConfig({ app, baseUrl: "http://example.com" });
 
-      expect(app.set).toBeCalledWith("API.BASE_URL", "http://example.com");
+      expect(app.set).toHaveBeenCalledWith(
+        "API.BASE_URL",
+        "http://example.com"
+      );
     });
 
     it("should set 'API.PATHS.SESSION", () => {
       setAPIConfig({ app, sessionPath: "/api/session" });
-
-      expect(app.set).toBeCalledWith("API.PATHS.SESSION", "/api/session");
+      expect(app.set).toHaveBeenCalledWith("API.PATHS.SESSION", "/api/session");
     });
 
-    it("should set 'API.PATHS.SESSION", () => {
+    it("should set 'API.PATHS.AUTHORIZATION", () => {
       setAPIConfig({ app, authorizationPath: "/api/authorization" });
 
-      expect(app.set).toBeCalledWith(
```

Comment:
Ah, that's probably mocha syntax, that's supported by Jest - https://jestjs.io/docs/expect#tohavebeencalledwitharg1-arg2-

Reply:
It's not about syntax but consistencty. There are two methods to do the same thing - `toBeCalledWith` is equivalent to `toHaveBeenCalledWith`. The linter recommends using one of them for consistency (it doesn't matter which one but the latter is recommended by the linter).